now_escape/components/Label/label.js

```diff
@@ -0,0 +1,58 @@
+import React, {Fragment} from 'react';
+import {Text, View, StyleSheet, TouchableOpacity} from 'react-native'
+
+export default function Label(props) {
+  const {height, width, borderRadius, type='', icon, text=''} = props;
+
+  return (
+    <Fragment>
+      {/* 타입별 스타일 및 이벤트 */}
+      {props.type == 'mainLabel' && (
+        <TouchableOpacity onPress={()=>alert('main label')}>
+          <View style={styles(width, height, borderRadius).main}>
```

Comment:
```suggestion
  const style = styles(width, height, borderRadius);

  return (
    <Fragment>
      {/* 타입별 스타일 및 이벤트 */}
      {props.type == 'mainLabel' && (
        <TouchableOpacity onPress={()=>alert('main label')}>
          <View style={style.main}>
```

조금 더 간단하게 적을 수 있을 것 같습니다!

추가로 styled-component도 고려해보면 좋을 것 같아요
https://dev-yakuza.posstree.com/ko/react-native/styled-components/
https://medium.com/@Devsteam.mobi/a-quick-performance-comparison-of-styled-components-vs-inline-styles-in-react-native-3125ae603d45